frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwareTitleDetailsTable/SoftwareTitleDetailsTableConfig.tsx

```diff
@@ -103,6 +116,7 @@ const generateSoftwareTitleDetailsTableConfig = (router: InjectedRouter) => {
               <ViewAllHostsLink
                 queryParams={{
                   software_version_id: cellProps.row.original.id,
+                  team_id: teamId,
```

Comment:
TODO: is this needed? convo with Jacob 2/13

Reply:
The answer is no, but doesn't do any harm leaving it in. We can even remove the `queryParams` altogether as long as it's the parent row that handles the click, which it is in each of these cases.